app/Http/Middleware/RequireScopes.php

```diff
@@ -47,8 +47,7 @@ protected function validateScopes(?Token $token, $scopes)
         }
 
         if (!$this->requestHasScopedMiddleware(request())) {
-            // use a non-existent scope; only '*' should pass.
-            if (!$token->can('invalid')) {
+            if (!$token->can('*')) {
```

Comment:
Shouldn't be changed from intentionally checking a scope that cannot be assigned; it's specifically like this because `can` just treats the presence of `*` on the token as a complete bypass and not a permission check - if there's an underlying issue, `*` will still pass while `invalid` will start failing which is the more desirable scenario.

Reply:
I'm not really following, what "underlying issue" could make `*` pass but not `invalid`?

if you meant like a programming error that causes `*` to not be the "bypass" scope anymore, it would be caught by tests already